src/index.ts

```diff
@@ -1,9 +1,9 @@
-import qs, { IStringifyOptions } from 'qs';
+import { stringify, StringifyOptions } from 'neoqs';
```

Comment:
Should be `neoqs/legacy`, as neoqs doesn't support CJS, but legacy subpath does

Reply:
Im not sure if current setup builds qs also in, based on my experience it prolly doesn't.

If it builds it in then its good, otherwise it should be switched to /legacy

Reply:
yes, I wish CI would trigger, but locally all tests pass

Reply:
tests won't catch this issue. Can you locally build and inspect whether the output file is importing qs or is it fully inlined?

Reply:
Good catch! Added noExternal from:  https://www.jsdocs.io/package/tsup#Options
```ts
/** Always bundle modules matching given patterns */
noExternal?: (string | RegExp)[];
```